tgui/packages/tgui/interfaces/AirlockElectronics.js

```diff
@@ -0,0 +1,111 @@
+// code\game\machinery\doors\airlock_electronics.dm
+import { useBackend } from '../backend';
+import { Button, Divider, Flex, Section } from '../components';
+import { Window } from '../layouts';
+import { AccessList } from './common/AccessList';
+import { Fragment } from 'inferno';
+
+export const AirlockElectronics = (props, context) => {
+  return (
+    <Window resizable>
+      <UnrestrictedAccess />
+      <Divider />
+      <ChooseAccess />
+    </Window>
+  );
+};
+
+const UnrestrictedAccess = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    unrestricted_dir,
+  } = data;
+  return (
+    <Section title="Access Control">
+      <Flex direction="column">
+        <Flex.Item bold mb={1}>
+          Unrestricted Access From:
+        </Flex.Item>
+        <Flex.Item>
+          <Button width={5.7}
```

Comment:
I would put these inside a `<Grid>` element and use `<Grid.Column>` to house each button, then add the `fluid` property to each button to make them fit the full width of the container, which would make them all equal width and fill a full space, looking a bit nicer in the UI

Reply:
I think I considered doing that a while ago but I went with `<Flex>` instead because it says this in the [Component Reference](https://github.com/ParadiseSS13/Paradise/blob/master/tgui/docs/component-reference.md#grid).
> Deprecated: This component is no longer recommended due to the variety of bugs that come with table-based layouts. We recommend using Flex instead.

I'll try it out anyway though, no reason not to.

Reply:
I've messed around with `<Grid>` a bit and I think I'll keep it as it is. While there *is* a blank space on the right side I personally prefer that to the buttons taking up the whole row. It just feels better, I suppose.